graphql/fetchUserGoalsCount.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from "@apollo/client";
+
+export const FETCH_USER_GOALS_COUNT = gql`
+query fetchUserGoalsCount($userId: String = "") {
+  user_goals_aggregate(where: { userId: { _eq: $userId } }) {
+    aggregate {
+      count
+    }
+  }
+}
+`;
+
+export type FetchGoalCountResponse = {
+  user_goals_aggregate: GoalAggregate;
+};
+
+export type GoalAggregate = {
+  aggregate: GoalAggregateNested;
+};
+
+export type GoalAggregateNested = {
+  count: number;
+};
+
+//commented out query that will pull onlly the number of goals that are due within 7 days.
```

Comment:
this is the new version of your query, this should stay in here

it's ok for us to have this file in this PR because it's just defining the query and the response type, it's not executing anything... the useQuery in the sidebar will actually execute every time someone uses Skillify, so that's an unnecessary performance hit if we aren't actually making use of it yet